python/cugraph/dask/traversal/mg_sssp.pxd

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from cugraph.structure.graph_primtypes cimport *
+from libcpp cimport bool
+
+
+cdef extern from "utilities/cython.hpp" namespace "cugraph::cython":
+
+    cdef void call_sssp[VT,WT](
```

Comment:
We'd better replace VT, ET, WT with vertex_t, edge_t, weight_t.

Reply:
updated